app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitSponsorshipsApiController.php

```diff
@@ -78,20 +88,20 @@ public function getAll($summit_id, $sponsor_id): mixed
         return $this->_getAll(
```

Comment:
@romanetar here we need to check is the current user has authz for the current summit

```
 $current_member = $this->resource_server_context->getCurrentUser();
            if (!is_null($current_member) && !$current_member->isSummitAllowed($summit))
                return $this->error403(['message' => sprintf("Member %s has not permission for this Summit", $current_member->getId())]);
```